Entities/Industry/Drill/Drill.as

```diff
@@ -87,6 +88,11 @@ bool canBePutInInventory( CBlob@ this, CBlob@ inventoryBlob )
 	return true;
 }
 
+bool canBePickedUp(CBlob@ this, CBlob@ byBlob)
+{
+	return (this.get_u8(heat_prop) < heat_max - heat_add * 1.5f);
```

Comment:
This expression is copy-pasted from `onTick` (`if (int(heat) >= heat_max - (heat_add * 1.5))`), factor those out in a single global constant as is done with other values

Reply:
done